build/scripts/Releasing.fsx

```diff
@@ -33,6 +33,11 @@ module Release =
 
             let year = sprintf "%i" DateTime.UtcNow.Year
 
+            let jsonDotNetVersion = 10
```

Comment:
Would have given this PR  ⭐️⭐️⭐️⭐️⭐️ had this been read from our csproj files. 

That said, submitting a PR that brings us back to `paket` soon so might have been waisted effort.